featuretools/primitives/base/primitive_base.py

```diff
@@ -51,3 +56,33 @@ def get_function(self):
 
     def get_filepath(self, filename):
         return os.path.join(config.get("primitive_data_folder"), filename)
+
+    def get_args_string(self):
+        strings = []
+        args = signature(self.__class__).parameters.items()
+        for name, arg in args:
+            # assert that arg is attribute of primitive
+            error = '"{}" must be attribute of {}'
+            assert hasattr(self, arg.name), error.format(arg.name, self.__class__.__name__)
+
+            # skip if *args or **kwargs
```

Comment:
should this be "skip if not"?

Reply:
The `*args` and `**kwargs`  would not be considered `POSITIONAL_OR_KEYWORD`. This comment could work as well.
```
# skip if not a standard argument (e.g. excluding *args and **kwargs)
```

Reply:
looks good